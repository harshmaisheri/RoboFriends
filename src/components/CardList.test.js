import React from "react";
import { shallow } from "enzyme";
import CardList from "./CardList";

const mockRobots = [
  {
    id: 1,
    name: "John Snow",
    username: "I Know Nothing",
    email: "johnsnow@got.com",
  },
];
const Wrapper = shallow(<CardList robots={mockRobots} />);

it("Expect to Render Card List", (done) => {
  expect(Wrapper.debug()).toMatchSnapshot();
  done();
});
