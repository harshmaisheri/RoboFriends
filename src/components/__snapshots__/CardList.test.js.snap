// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect to Render Card List 1`] = `
"<div>
  <Card id={1} name=\\"John Snow\\" email=\\"johnsnow@got.com\\" />
</div>"
`;
